maven.gradle

```diff
@@ -1,12 +1,27 @@
-import com.infinum.maven.SonatypeConfiguration
-
-SonatypeConfiguration.metaClass.constructor = { ->
-    def constructor = SonatypeConfiguration.class.getConstructor()
-    def instance = constructor.newInstance()
-    instance.load()
-    instance
+def properties = new Properties()
+def propsFile = file("$rootDir/publish.properties")
```

Comment:
We don't have any examples for this file, what should be in it? 🤔

Reply:
It's described in the handbook, that's why I saw no point in explaining here as its not just a coupe of lines.

Reply:
TBH I'm not against putting all that info here as well, it's easier to delete 20 lines of docs than it is to open a handbook and search for it. Though I also agree it is something that the handbook section on library publishing should contain.